### PR TITLE
[DFFS-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by TunrayoIlawole**
Added database for a transportation company.

### CODE REVIEW

File: submissions/motunrayoIlawole/Database/transportation.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
+CREATE DATABASE travel_corp;
+
+USE travel_corp;
+
+/* Create table for registered passengers */
+CREATE TABLE registered_passengers (
+	id INT NOT NULL AUTO_INCREMENT,
+    full_name VARCHAR(100) NOT NULL,
+    sex VARCHAR(7) NOT NULL,
+    age INT,
```
Comment (Ireti203) [inline submissions/motunrayoIlawole/Database/transportation.sql:10]: Kindly uses "varchar" here

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/motunrayoIlawole/Database/transportation.sql (hunk 1 above)
Comment (TunrayoIlawole) [reply submissions/motunrayoIlawole/Database/transportation.sql:10]: For the age?

Comment (adaugochi) [reply submissions/motunrayoIlawole/Database/transportation.sql:10]: Yes

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
@TunrayoIlawole are you such you made any changes? it didn't reflect on your PR

**Pull request closed without merging**